ndb compilation fails with -Werror on gcc 9.2.1 / glibc 2.30

Starting with commit 49d43a334572199088ecc24ce6fc0e86c8699d2e, I'm getting the following error:

> backend/ndb/rpmxdb.c: In function ‘rpmxdbDelAllBlobs’:
> backend/ndb/rpmxdb.c:966:5: error: ignoring return value of ‘ftruncate’, declared with attribute warn_unused_result [-Werror=unused-result]
>  966 |     ftruncate(xdb->fd, xdb->pagesize);
>      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> cc1: all warnings being treated as errors

Guess CI is running on something older since this didn't come show up there.

--- a/lib/backend/ndb/rpmxdb.c
+++ b/lib/backend/ndb/rpmxdb.c
@@ -963,7 +963,9 @@ int rpmxdbDelAllBlobs(rpmxdb xdb)
 	rpmxdbUnlock(xdb, 1);
 	return RPMRC_FAIL;
     }
-    ftruncate(xdb->fd, xdb->pagesize);
+    if (ftruncate(xdb->fd, xdb->pagesize)) {
+	/* ftruncate failed, but that is not a problem */
+    }
     rpmxdbUnlock(xdb, 1);
     return RPMRC_OK;
 }

